plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4507,6 +4507,64 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
                 }
                 return new ElementTag(chiseledBookshelf.getSlot(input.toVector()) + 1);
             });
+
+            // <--[tag]
+            // @attribute <LocationTag.crafter_disabled_slots>
+            // @returns ListTag
+            // @mechanism LocationTag.crafter_disabled_slots
+            // @group world
+            // @description
+            // Returns the disabled slots of a crafter block from left to right, top to bottom.
```

Comment:
I'm not sure if the order of the list is something we should be guaranteeing here? Realistically any usage should depend on what the actual slots are, not where they are in the list.